### PR TITLE
Vector init in qt5

### PR DESCRIPTION
**Pull request by outofculture**
`Vector.__init__` improvements

 * add docstring
 * fix handling of QVector3D args (cannot list() them)
 * refactor to no longer need return statements

Also add Pa to the list of units.

### CODE REVIEW

**Comment by outofculture** [comment]
Rebased onto master; turns out the refactor had happened once before, but this one is more accurate.

File: pyqtgraph/Vector.py [hunk 1]
```diff
@@ -12,35 +12,37 @@ class Vector(QtGui.QVector3D):
     """Extension of QVector3D which adds a few helpful methods."""
     
     def __init__(self, *args):
+        """
+        Handle additional constructions of a Vector
+
+        ==============  ================================================================================================
+        **Arguments:**
+        *args*      Could be any of:
```
Comment (ixjlyons) [inline pyqtgraph/Vector.py:20]: Vector isn't currently in the docs, but there should be few more spaces introduced before "Could". It should align with the `=` after the gap in the table borders. Please also add an empty line after this one - I'm not sure exactly why but RST/sphinx seems to prefer it this way for bulleted lists.

File: pyqtgraph/Vector.py [hunk 2]
```diff
@@ -12,35 +12,37 @@ class Vector(QtGui.QVector3D):
...
+                     * 3 numerics
+                     * 2 numerics (`0` assumed for z)
+                     * Either of the previous in a list-like collection
+                     * 1 QSizeF (`0` assumed for z)
+                     * 1 QPointF (`0` assumed for z)
+                     * Any other valid QVector3D init args.
+        ----------------------------------------------------------------------------------------------------------------
```
Comment (ixjlyons) [inline pyqtgraph/Vector.py:27]: I think this line can be removed.

File: pyqtgraph/Vector.py [hunk 3]
```diff
@@ -12,35 +12,37 @@ class Vector(QtGui.QVector3D):
     """Extension of QVector3D which adds a few helpful methods."""
     
     def __init__(self, *args):
+        """
+        Handle additional constructions of a Vector
+
+        ==============  ================================================================================================
+        **Arguments:**
+        *args*      Could be any of:
+                     * 3 numerics
```
Comment (ixjlyons) [inline pyqtgraph/Vector.py:21]: It's probably obvious, but maybe add a parenthetical `(x, y, and z)` at the end of this line.

File: pyqtgraph/Vector.py (hunk 1 above)
Comment (outofculture) [reply pyqtgraph/Vector.py]: Thanks; I was bumbling about for the rules there.

**Comment by outofculture** [comment]
I've acted on all the comments.

**Comment by ixjlyons** [comment]
Looks good. Thanks @outofculture

**Pull request merged**